sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_base_test.hpp

```diff
@@ -43,7 +43,7 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Keys { nam
         Azure::Response<T>& response,
         Azure::Core::Http::HttpStatusCode expectedCode = Azure::Core::Http::HttpStatusCode::Ok)
     {
-      auto rawResponse = response.ExtractRawResponse();
+      auto rawResponse = std::move(response.RawResponse);
```

Comment:
```suggestion
      auto& rawResponse = response.RawResponse;
```
Then update line 49 to `rawResponse.` instead of `rawResponse->`.  I made this change to my latest keyvault PR https://github.com/Azure/azure-sdk-for-cpp/pull/1967/files#diff-bc3dcb1e806ad216a89560228cedd0cb8d51838217308297720d24c90c2d7f94L46

Reply:
Can you please explain why we want to do this?

Reply:
I think I get it. You don't want `CheckValidResponse` to extract ownership of the `RawResponse`, because then the caller can't keep using it from the `Response<T>` passed-in.

Reply:
> Then update line 49 to `rawResponse.` instead of `rawResponse->`.

This wasn't required.